cypress/support/pages/course/pageEditCourse.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Edit_Course {
+  static #courseToEdit = '[aria-label="Kurs Mathe"]'
+  static #pageTitle = '[id="page-title"]'
+  static #dropDownCourse = '[class="btn btn-sm dropdown-toggle btn-course-dropdown"]'
+  static #btnCourseEdit = '[class="dropdown-item btn-course-edit"]'
+  static #selectClassesInput = '#classId_chosen > ul > li > input'
```

Comment:
selectClass, singular should be sufficient, since you only add one class, or?
Same applies for the other variables as well, I'd say.

Reply:
I wrote classes as the function of the considered field is also for selecting more than one class.